esphome/const.py

```diff
@@ -437,10 +437,19 @@
 CONF_PIXEL_MAPPER = "pixel_mapper"
 CONF_PLATFORM = "platform"
 CONF_PLATFORMIO_OPTIONS = "platformio_options"
+CONF_PM_0_3um = "pm_0_3um"
+CONF_PM_0_5um = "pm_0_5um"
 CONF_PM_1_0 = "pm_1_0"
+CONF_PM_1_0_STD = "pm_1_0_std"
+CONF_PM_1_0um = "pm_1_0um"
 CONF_PM_10_0 = "pm_10_0"
+CONF_PM_10_0_STD = "pm_10_0_std"
+CONF_PM_10_0um = "pm_10_0um"
 CONF_PM_2_5 = "pm_2_5"
+CONF_PM_2_5_STD = "pm_2_5_std"
+CONF_PM_2_5um = "pm_2_5um"
 CONF_PM_4_0 = "pm_4_0"
+CONF_PM_5_0um = "pm_5_0um"
```

Comment:
```suggestion
CONF_PM_0_3UM = "pm_0_3um"
CONF_PM_0_5UM = "pm_0_5um"
CONF_PM_1_0 = "pm_1_0"
CONF_PM_1_0_STD = "pm_1_0_std"
CONF_PM_1_0UM = "pm_1_0um"
CONF_PM_10_0 = "pm_10_0"
CONF_PM_10_0_STD = "pm_10_0_std"
CONF_PM_10_0UM = "pm_10_0um"
CONF_PM_2_5 = "pm_2_5"
CONF_PM_2_5_STD = "pm_2_5_std"
CONF_PM_2_5UM = "pm_2_5um"
CONF_PM_4_0 = "pm_4_0"
CONF_PM_5_0UM = "pm_5_0um"
```

All keys in `conf.py` are uppercase (because they're constants)

Reply:
updated